docs/references/cmcspec.mdx

```diff
@@ -0,0 +1,552 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Cycles Minting Canister (CMC) API
+
+<MarkdownChipRow labels={["Reference"]} />
+
+The Cycles Minting Canister (CMC) is a system canister on the Internet Computer responsible for converting ICP tokens into **cycles**, tracking the total cycles minted, and providing configuration and exchange rate data. It supports:
+
+-   Topping up existing canisters.
+-   Creating new canisters.
+-   Minting and depositing minted cycles into cycle ledger-managed accounts.
```

Comment:
"cycle ledger-managed" has hyphenation that is confusing.

Perhaps "into accounts managed by the Cycles Ledger" would be clearer?